src/mscorlib/src/System/TimeZoneInfo.Win32.cs

```diff
@@ -9,8 +9,12 @@
 using System.Security;
 using System.Text;
 using System.Threading;
+
 using Microsoft.Win32;
 using Microsoft.Win32.SafeHandles;
+using REG_TZI_FORMAT = Microsoft.Win32.Win32Native.RegistryTimeZoneInformation;
```

Comment:
would it be clearer to use the type names (just crop off the namespace) instead of the struct names? Seems the point of those types is to wrap the structs.

Reply:
This temporary glue. There will be just the `REG_TZI_FORMAT` structure once I am done reconciling the platform-specific parts. RegistryTimeZoneInformation will be gone.

`REG_TZI_FORMAT` matches the coding convention that they say to use the official SDK name and casing.

Reply:
Agree. Back then when I talked with @tarekgh about that matter we discovered that we can get rid of the `RegistryTimeZoneInformation` type and just use the interop struct.